notify/impl.go

```diff
@@ -573,8 +573,13 @@ func (n *Hipchat) Notify(ctx context.Context, as ...*types.Alert) error {
 		msg = tmplText(n.conf.Message)
 	}
 
+	from := tmplText(n.conf.From)
+	if len(from) > 64 {
```

Comment:
We're going to have a lot of these, I'd suggest breaking this out to a function.